includes/wp-admin/class-settings.php

```diff
@@ -244,6 +244,26 @@ public static function register_settings() {
 				'default'     => null,
 			)
 		);
+
+		\register_setting(
+			'activitypub_blog',
+			'activitypub_blog_user_also_known_as',
+			array(
+				'type'              => 'array',
+				'description'       => \__( 'Also Known As', 'activitypub' ),
+				'default'           => array(),
+				'sanitize_callback' => function ( $value ) {
+					if ( ! is_array( $value ) ) {
+						$value = explode( PHP_EOL, $value );
+					}
+
+					$value = array_filter( array_map( 'trim', $value ) );
+					$value = array_filter( array_map( 'esc_url', $value ) );
```

Comment:
Not yet sure whether it should be `esc_url_raw` or not, since we're about to save it (`db` context) and not display it. We could probably move the `array_filter()` to the return line and only do it once?